SPECS/python-werkzeug/python-werkzeug.spec

```diff
@@ -33,22 +40,37 @@ Werkzeug started as simple collection of various utilities for WSGI applications
 %prep
 %autosetup -n werkzeug-%{version} -p1
 
+%generate_buildrequires
+%pyproject_buildrequires %{?with_tests:-t}
+
 %build
-%py3_build
+%pyproject_wheel
 
 %install
-%py3_install
+%pyproject_install
+%pyproject_save_files werkzeug
 
 %check
-pip3 install pytest hypothesis
-LANG=en_US.UTF-8 PYTHONPATH=./  python3 setup.py test
-
-%files -n python3-werkzeug
+pip3 install tox==4.6.3 tox-current-env
+pip3 install -r requirements/tests.txt
+# The default %%tox macro causes some test failures seemingly due to the way werkzeug invokes additional python processes during the
+# testing process, so tox is invoked manually here.
+# This is a known limitation of tox-current-env. See https://pypi.org/project/tox-current-env/ under "Do not rely on virtualenv details"
+# for more information.
+tox -q --recreate -e %{toxenv}
+
+%files -n python3-werkzeug -f %{pyproject_files}
 %defattr(-,root,root)
-%license LICENSE.rst
-%{python3_sitelib}/*
+%doc README.rst
+%doc CHANGES.rst
```

Comment:
We still want a `%license` macro installing some file explaining the licensing for the package. If the source no longer has anything suitable for that purpose (sometimes licensing info is included in a readme file and that's also OK), then please create a `LICENSE.PTR` file locally and explain the licensing there in a sentence or two. Search for `LICENSE.PTR` in other specs to get an example but usually we end up going through some of the source's code or documentation, find licensing information there and then mention how we found the license in that new file.

Reply:
Misread the documentation for the `%{pyproject_files}` macro. Will re-add.